core/merge.js

```diff
@@ -484,6 +484,10 @@ class Merge {
         metadata.markAsUnsyncable(side, src)
         metadata.markAsNew(dst)
         metadata.markSide(side, dst)
+      } else if (src.sides && !src.sides[side] && src.sides[otherSide(side)]) {
+        metadata.markAsUnsyncable(otherSide(side), src)
+        metadata.markAsNew(dst)
+        metadata.markSide(otherSide(side), dst)
```

Comment:
Looks similar to the other branch...

```js
const singleSide = detectSingleSide(src)
if (singleSide) {
  convertToDestinationAddition(src, dst, otherSide(singleSide))
  // or maybe reverse args, same as move* methods? e.g.
  convertToDestinationAddition(otherSide(singleSide), dst, src)
}

const sideNames = new Set(['local', 'remote'])

const detectSingleSide = doc => {
  if (doc.sides) {
    for (const sideName of sideNames) {
      if (doc.sides[sideName] && !doc.sides[otherSide(sideName)]) {
        return sideName
      }
    }
  }
}

const convertToDestinationAddition = (src, dst, side) => {
  metadata.markAsUnsyncable(otherSide(side), src)
  metadata.markAsNew(dst)
  metadata.markSide(side, dst)
}
```

I wonder whether renaming `markAsUnsyncable()` to `forget()` would be clearer. Do we even need to pass it a side? Out of the scope of this PR anyway.